bin/fixmystreet.com/process_update_file

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env perl
+#
+# Takes a JSON file with update details and uses them to populate
+# missing updates
+
+use strict;
+use warnings;
+use v5.14;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../setenv.pl";
+}
+
+use Getopt::Long::Descriptive;
+use FixMyStreet::Script::ProcessUpdateFile;
+
+my ($opts, $usage) = describe_options(
+    '%c %o',
+    ['file|f:s', 'file containing updates' ],
```

Comment:
I think if you put f=s and b=s, and { required => 1}, that will make it mandatory and require the argument, so you don't need to do the file/body checking below.